lib/provider.coffee

```diff
@@ -299,8 +299,9 @@ lineEndsWithSemicolon = (bufferPosition, editor) ->
   line = editor.lineTextForBufferRow(row)
   /;\s*$/.test(line)
 
-hasScope = (scopesArray, scope) ->
-  scopesArray.indexOf(scope) isnt -1
+hasScope = (scopesArray, scope, checkEmbedded = false) ->
+  scopesArray.indexOf(scope) isnt -1 or
+    (checkEmbedded and scopesArray.indexOf(scope + '.embedded.html') isnt -1)
```

Comment:
:art: `(checkEmbedded and scopesArray.indexOf("#{scope}.embedded.html") isnt -1)`